src/Carrier.php

```diff
@@ -57,6 +62,7 @@ class Carrier
         self::CODE_UPS => 'UPS',
         self::CODE_USPS => 'USPS',
         self::CODE_LASER_SHIP => 'LaserShip',
+        self::CODE_ONTRAC => 'OnTrac',
```

Comment:
Alpha sort is helpful for readability as the list of supported carriers gets longer.

Can you reorder these array elements to be in alphabetical order? E.g., LaserShip should come after FedEx, followed by OnTrac.